### ISSUE TITLE
WAApplication>>sessionsDo: iterates over document handlers as well

### ISSUE THREAD

**Issue by daniels220** [SeasideSt/Seaside#1318]
I would expect #sessionsDo: to iterate over only sessions (instances of WASession), but the implementation makes no distinction so document handlers are included as well. I bet when the implementation was added per #416 document handlers didn't exist.

**Comment by marschall** [comment]
It shouldn't. The only request handlers in an application should be sessions. The document handlers should be stored in the session, see #709 and #710.

**Comment by daniels220** [comment]
Huh. At this point I can't seem to reproduce the issue in my codebase—I know I ran across it in a particular spot, but when I call `#sessions` there now I indeed get only sessions. I don't _think_ I changed anything else that would matter—it would certainly be believable that I was just adding a document handler in the wrong place somewhere, but I certainly haven't removed such a thing since I made this issue. In any case, I guess it's fine to just close this then.
